ansible/roles/ansible-postgres_patroni/defaults/main.yml

```diff
@@ -0,0 +1,16 @@
+---
+# defaults file for ansible-postgres_patroni
+#patroni .yaml config
+Postgres_cluster_name: postgresql-prod
+
+# users admin password
+admin_password: admin
```

Comment:
Can we prefix these default vars with the role name so that there is no conflict with other role vars during overrides?
Example - 
`postgres_patroni_admin_password: admin`
`postgres_patroni_replication_username: replicator`

The reason behind this ask is so that a user can identify to which role a variable belongs and also avoid collisions with other role default vars.

Please also make the change in other template files to use this new var after implementing this change.

Reply:
comments updated